### ISSUE TITLE
2 classes training gives solid black image

### ISSUE THREAD

**Issue by sakherg** [Attila94/refinenet-keras#11]
Thank you for sharing your code!
I am trying to train my own dataset which includes only 2 classes (hand rgb:255 255 255, background rgb:0,0,0). I did not use the pre-trained file. I arrange the dataset as you instruct but the result during the train is a black image. I trained the model more than 100 epochs and the loss is around 0.73130.
Any ideas or suggestions?

**Comment by Attila94** [comment]
This is unfortunately a known issue that pops up every now and then which I never managed to resolve. The model most likely gets stuck in some local minimum, perhaps due to a class imbalance in the training data.

In the meantime I switched to PyTorch where I didn't encounter this issue and is a much nicer framework in general imho.

**Comment by sakherg** [comment]
Thank you for your response. I solved the problem by make some modifications:
1- change the optimization function to "RMSprop".
2- use focal loss function as in the link: https://github.com/umbertogriffo/focal-loss-keras

Thank you!
